web calculator/web calculator/styles.css

```diff
@@ -0,0 +1,7 @@
+.body{
+    background-color: cornflowerblue
+}
+
+.container{
+    
+}
```

Comment:
Looks like your started to add more styles but didn't. Always delete code you don't end up using so it doesn't clutter up the code base.